sycl/test/basic_tests/image_accessor.cpp

```diff
@@ -0,0 +1,44 @@
+// RUN: %clang -std=c++11 %s -o %t1.out -lstdc++ -lOpenCL -lsycl
+// RUN: env SYCL_DEVICE_TYPE=HOST %t1.out
+// RUN: %clang -std=c++11 -fsycl %s -o %t2.out -lstdc++ -lOpenCL -lsycl
+// RUN: env SYCL_DEVICE_TYPE=HOST %t2.out
+// RUN: %CPU_RUN_PLACEHOLDER %t2.out
+// RUN: %GPU_RUN_PLACEHOLDER %t2.out
+// RUN: %ACC_RUN_PLACEHOLDER %t2.out
+//==------------------- d;sjfas;fjs;fkjsa;dfjask;ldfja;slfjas---------------==//
```

Comment:
//==------------------- d;sjfas;fjs;fkjsa;dfjask;ldfja;slfjas---------------==//

How about meaningful comment in English?

Reply:
forgot to delete the file